src/PrestaShopBundle/Resources/config/services/adapter/api_access.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  _defaults:
+    public: true
```

Comment:
Maybe we can simplify this and set by defaults autowire & autoconfigure to true.
What do your think?